src/App.js

```diff
@@ -7,13 +7,13 @@ import Categories from './components/Categories';
 
 function App() {
   return (
-    <>
+    <div className="App">
       <NavBar />
       <Routes>
         <Route exact path="/" element={<Books />} />
         <Route path="/categories" element={<Categories />} />
```

Comment:
Please ensure that the styling is EXACTLY the same as the one provided on [Zeplin](https://zeplin.io/)Styling is EXACTLY the same as the one provided on [Zeplin](https://zeplin.io/) according to the project requirement.

Your Implementation | Design
:---:|:---:
![image](https://user-images.githubusercontent.com/10332499/167264845-6ed547cd-7ece-4aa0-8eb3-90a0ed3ce988.png) | ![image](https://user-images.githubusercontent.com/10332499/167264932-e435504f-bb00-4988-80eb-97b61c6ef97b.png)